### DISCUSSION TITLE
Can pdf clipping be done in chrome

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#82] [Q&A] [answered]
I don't think its possible to clip from pdf in chrome currently. Can that be done ? If yes, snap from selection could be done for pdf as mentioned in [this discussion](https://github.com/buggyj/tiddlyclip-plugin/discussions/76#discussion-3876575)

**Comment by buggyj** [comment, marked as answer]
I hardly ever use tiddlyclip with chrome. Chrome is changing the version of its plugins api so the current version of tiddlyclip will need to be updated - probably within12 months.

**Reply by arunnbabu81** [reply]
> I hardly ever use tiddlyclip with chrome. Chrome is changing the version of its plugins api so the current version of tiddlyclip will need to be updated - probably within12 months.

I asked this because in chrome we could  use cross hairs to select snaps in pdfs which is not possible in Firefox as of yet.

**Comment by arunnbabu81** [comment]
@buggyj Any plans for pdf annotation support in the chrome add on

**Reply by buggyj** [reply]
Soon the tiddlyclip chrome add on is going to stop working...

It is now possible to copy images from the paste buffer in firefox, so it would be possible to use the firefox screen capture tool, copy the the paste buffer and then using tiddlyclip to send the contents of the buffer. I will need to update tiddlyclip addon for this.

**Reply by arunnbabu81** [reply]
Does it mean that it will it be possible to snap images from pdf in Firefox after the update ? How long it may take for the  next update ?

**Reply by buggyj** [reply]
yes it means that. I do not know when I will be able to put the change in.
